flight/PiOS/STM32F10x/pios_flash_internal.c

```diff
@@ -62,6 +62,11 @@ static struct pios_internal_flash_dev *PIOS_Flash_Internal_alloc(void)
 	return(flash_dev);
 }
 
+
+void FLASH_IRQHandler() {
```

Comment:
:+1: 
It would be neat to have a DEFINE that causes separate handlers like this to be built for all the unused/error IRQs, otherwise just weak link them all to a single default handler. I regularly find myself splitting them all out when debugging crashes that we encounter from time to time.

Reply:
We could have a single generic one that grabs IPSR and puts it into a variable for inspection.

Reply:
You can dump IPSR out easy enough already, but then you need a table handy to look it up. :stuck_out_tongue: